dependencies.properties

```diff
@@ -25,7 +25,7 @@ version.gax_httpjson=0.81.1-SNAPSHOT
 # with the sources.
 version.com_google_protobuf=3.15.2
 version.google_java_format=1.1
-version.io_grpc=1.36.0
+version.io_grpc=1.37.0
```

Comment:
optional but you might try to send this upgrade first as a separate PR

Reply:
done